composer.json

```
@@ -20,12 +20,12 @@
         }
     ],
     "require": {
-        "php": ">=7.3,>=8.0",
-        "laravel/framework": "^8.0"
+        "php": ">=7.3,>=8.2",
+        "laravel/framework": "^8.0,^9.0"
```

This doesn't stop support of 8.0 and 8.1 does it?

```suggestion
        "php": "~7.3|~8.0.0|~8.1.0|~8.2.0",
        "laravel/framework": "^8.0|^9.0"
```